common/src/main/java/com/wynntils/handlers/item/ItemHandler.java

```diff
@@ -104,13 +108,35 @@ private void onItemStackUpdate(ItemStack existingItem, ItemStack newItem) {
             return;
         }
 
+        // I don't actually know a clean way to get 2 returns from one method when one is a boolean for an if condition
+        // or somehow not repeat a lot of code and seperate it to 2 methods.
+        int[] shinyStatLineIndex = {-1};
```

Comment:
You can just use pairs, although if you remove the event, this is not needed atm.